tap-setup-scripts/src/cloud-init.sh

```diff
@@ -62,8 +62,8 @@ systemctl start amazon-cloudwatch-agent.service
 systemctl status amazon-cloudwatch-agent.service
 echo "Installing kubectl..."
 pushd /tmp
-aws s3 cp --no-progress "s3://amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl" ./kubectl
-aws s3 cp --no-progress "s3://amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl.sha256" ./kubectl.sha256
+curl --retry 5 -o ./kubectl "https://s3.us-west-2.amazonaws.com/amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl"
```

Comment:
*question*: Does this fail on error and does it follow redirects? Is this maybe implicitly handled by `--retry`?

Usually I'd run `curl -LsSf`, so that:
- it is silent
- follows redirects
- fails on (most common server) errors
- reports errors

I am most worried about curl not failing when there is some issue, but our script just ignoring that.

Reply:
i dont think retry follows the redirects. if the download doesnt work for some reason the next steps in the script will error but then it is not so easy to troubleshoot what happened. 
I agree with the suggested change and will test it out